ts/features/services/home/components/FeaturedInstitutionCard.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  Avatar,
+  H6,
+  IOColors,
+  IOVisualCostants,
+  VSpacer
+} from "@pagopa/io-app-design-system";
+import { OrganizationFiscalCode } from "@pagopa/ts-commons/lib/strings";
+import React from "react";
+import { Dimensions, StyleSheet, View } from "react-native";
+import Placeholder from "rn-placeholder";
+import { WithTestID } from "../../../../types/WithTestID";
+import { logoForInstitution } from "../utils";
+
+export type FeaturedInstitutionCardProps = WithTestID<{
+  id: string;
+  name: string;
+  accessibilityLabel?: string;
+  isNew?: boolean;
+  onPress?: (id: string) => void;
+}>;
+
+export const CARD_WIDTH =
+  Dimensions.get("window").width - IOVisualCostants.appMarginDefault * 2;
+
```

Comment:
What about adding a summary of visual attributes at the top, using the correct types when available?
```suggestion

const cardPaddingHorizontal: IOSpacingScale = 12;
const cardBorderRadius: number = 8;

```

Reply:
aa1955420